arm-runner/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+  # chia-runner/tasks/main.yml
+  - name: Clone runner code and unzip
+    shell: mkdir actions-runner && cd actions-runner && curl -O -L https://github.com/actions/runner/releases/download/v2.273.5/actions-runner-linux-x64-2.273.5.tar.gz && tar xzf ./actions-runner-linux-x64-2.273.5.tar.gz
```

Comment:
Would be better to break this down into the actual ansible modules for creating directories, downloading files, etc, so that ansible can be smart about not downloading things when we already have the stuff installed